base/_base.scss

```diff
@@ -0,0 +1,28 @@
+@charset "UTF-8";
```

Comment:
Code climate is giving an error with the message "Prefer single quoted strings". I think that is a little too nit picky. Would you object to disabling that error check?